Source/View Controllers/TCCProfileViewController.swift

```diff
@@ -122,34 +128,52 @@ class TCCProfileViewController: NSViewController {
     @IBOutlet weak var uploadButton: NSButton!
     @IBOutlet weak var addAppleEventButton: NSButton!
     @IBOutlet weak var removeAppleEventButton: NSButton!
-
-    @IBOutlet weak var recordButton: NSButton!
-
-    @IBAction func recordPressed(_ sender: NSButton) {
-        canEdit = !canEdit
-        if canEdit {
-            recordButton.title = "Record"
-        } else {
-            recordButton.title = "Stop"
-        }
-    }
+    @IBOutlet weak var removeExecutableButton: NSButton!
+    @IBOutlet weak var authorizationKeySwitch: NSSwitch!
 
     @IBAction func addToProfile(_ sender: NSButton) {
         promptForExecutables {
             self.model.selectedExecutables.append($0)
         }
     }
 
-    //  Binding currently deletes at index
-    @IBAction func removeButtonPressed(_ sender: NSButton) {
-    }
-
     @IBAction func addToExecutable(_ sender: NSButton) {
         promptForExecutables {
             self.insertIntoAppleEvents($0)
         }
     }
 
+    private func toggleAuthorizationKey(theSwitch: NSSwitch, showAlert: Bool) {
+        switch theSwitch.state {
+        case .on:
+            if model.usingLegacyAllowKey && showAlert {
+                let message = """
+                Enabling Big Sur Compatibility will require the profile to be installed on macOS versions Big Sur (11.0) or greater.
```

Comment:
Think it'd be worth using `10.16` here instead? Or `10.16/11.0`?

```suggestion
                Enabling Big Sur Compatibility will require the profile to be installed on macOS versions Big Sur (10.16) or greater.
```

Reply:
Yeah, I'm not sure what to do there. I'm not a fan of Apple's conditional versioning for Big Sur.

Reply:
Maybe we could just omit the number and say "on macOS Big Sur or later".